administration/src/bp-modules/stores/StoresImportController.tsx

```diff
@@ -0,0 +1,76 @@
+import { NonIdealState } from '@blueprintjs/core'
+import React, { ReactElement, useContext, useState } from 'react'
+import { useNavigate } from 'react-router-dom'
+
+import { WhoAmIContext } from '../../WhoAmIProvider'
+import getMessageFromApolloError from '../../errors/getMessageFromApolloError'
+import { Role, useImportAcceptingStoresMutation } from '../../generated/graphql'
+import { ProjectConfigContext } from '../../project-configs/ProjectConfigContext'
+import { StoreFieldConfig } from '../../project-configs/getProjectConfig'
+import { useAppToaster } from '../AppToaster'
+import { AcceptingStoreEntry } from './AcceptingStoreEntry'
+import StoresButtonBar from './StoresButtonBar'
+import StoresCSVInput from './StoresCSVInput'
+import StoresTable from './StoresTable'
+
+const StoresImportController = (): ReactElement => {
+  const { role } = useContext(WhoAmIContext).me!
+  const storeManagement = useContext(ProjectConfigContext).storeManagement
+  if (role !== Role.ProjectStoreManager || !storeManagement.enabled) {
```

Comment:
I guess we also need this check for /stores route, because currently I can login, for example, as a region admin and open http://localhost:3000/stores

Reply:
yes definitly. i thought i catched that